core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -2597,6 +2605,22 @@ public void setAttribute(AlluxioURI path, SetAttributeOptions options)
   }
 
   /**
+   * Checks whether the owner belongs to the group.
+   *
+   * @param owner the owner to check
+   * @param group the group to check
+   * @throws FailedPreconditionException if owner does not belong to group
+   */
+  private void checkUserBelongsToGroup(String owner, String group)
+      throws IOException {
```

Comment:
@aaudiber 
- should we also put `FailedPreconditionException` in the signature? 
- should we also javadoc `IOE`?